spec/requests/casa_orgs_spec.rb

```diff
@@ -13,6 +13,11 @@
 # sticking to rails and rspec-rails APIs to keep things simple and stable.
 
 RSpec.describe '/casa_orgs', type: :request do
+  before do
+    skip "None of these currently do authorization, and I think that might be necessary. \
```

Comment:
I amok with skipping these for now because we aren't working with mutliple casa orgs for MVP. 
Eventually we will want to use the Devise test helpers like in admin_adds_new_case_spec.rb
```
sign_in admin # devise helper to sign in as the user saved in the variable "admin"
```

Reply:
I figured that was the case but wasn't sure of the specifics, so I didn't want to implement it incorrectly.